src/util/intOps.ml

```diff
@@ -42,7 +42,7 @@ sig
   val compare : t -> t -> int
   val equal : t -> t -> bool
   val hash : t -> int
-  val top_range : t -> t -> bool
+  val top_range : t -> t -> bool (* TODO: unused *)
```

Comment:
Same here, I am not so sure what the intended semantics is here

Reply:
Looking at the implementations, it seems to be for checking if the two arguments are `min_int` and `max_int` respectively. Again I guess we might've used that pre-bigints to do some special treatment, but that seems specific to intervals anyway.